about.html

```diff
@@ -0,0 +1,250 @@
+
+      <div class="latest-news-underline"></div>
+
+      <p class="latest-news-para">Take a look at our latest 2022 Porsche Model making waves and driving celebrities
+        crazy.
+      </p>
+
+      <div class="overlay">
+        <div class="overlay-content">
+          <h2 class="overlay-title">Panamera</h2>
+          <p class="overlay-para">
+            The Porsche Panamera is, as far as anyone knows
+          </p>
+        </div>
+      </div>
+
+      <div class="overlay-two">
+        <div class="overlay-content-two">
+          <h2 class="overlay-title-two">Cayenne</h2>
+          <p class="overlay-para-two">
+            The Porsche Cayenne is, as far as anyone knows
+          </p>
+        </div>
+      </div>
+
+      <!-- OVERLAY FOR DESKTOP -->
+
+      <div class="overlay-desktop">
+        <div class="overlay-desk">
+          <div class="overlay-content-desk">
+            <h2 class="overlay-title-desk">Panamera</h2>
+            <p class="overlay-para-desk">
+              The Porsche Panamera is, as far as anyone knows
+            </p>
+          </div>
+        </div>
+
+        <div class="overlay-two-desk">
+          <div class="overlay-content-two-desk">
+            <h2 class="overlay-title-two-desk">Cayenne</h2>
+            <p class="overlay-para-two-desk">
+              The Porsche Cayenne is, as far as anyone knows
+            </p>
+          </div>
+        </div>
+      </div>
+
+    </div>
+  </section>
+
+  <!-- PARTNERS SECTION STARTS -->
+  <section class="partners" id="partners-desktop">
+
+    <h2 class="partners-heading">Our Partners</h2>
+    <div class="partners-underline"></div>
+
+    <div class="partners-cards">
+
+      <div class="partners-card">
+        <img src="images/partners/totalenergies.png" class="total-energy" />
+      </div>
+
+      <div class="partners-card">
+        <img src="images/partners/netflix.png" class="netflix" />
+      </div>
+
+      <div class="partners-card">
+        <img src="images/partners/coke.png" class="Coca Cola" />
+      </div>
+
+      <div class="partners-card">
+        <img src="images/partners/facebook-1.png" class="Facebook" />
+      </div>
+
+      <div class="partners-card">
+        <img src="images/partners/google-2.png" class="Google" />
+      </div>
+
+    </div>
+
+  </section>
+
+  <!-- PARTNERS SECTION ENDS -->
+
+  <!-- FOOTER SECTION STARTS -->
+  <footer class="footer-about">
+
+    <div class="footer-content-about">
+      <div class="footer-content-logo-about">
+        <a href="/" class="footer-logo-about">Porsche</a>
+        <p class="footer-text-about">Porsche Club of America.</p>
+      </div>
+
+      <div class="footer-content-links-about">
+        <ul>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+        </ul>
+      </div>
```

Comment:
- [ ] In the desktop version of the about page, the footer is dark and in the mobile version it's light, kindly update it to follow the design guidelines.
- [ ] As I noted previously, kindly remove the navigation links and keep only the logo and the text as shown in the design guidelines 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/203853926-43ee28cb-e75f-47e9-ad14-24848c4b7670.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/203853971-afbcab25-b074-46c8-9afe-8a8bb18dc111.png)